test/jdk/java/nio/Buffer/CharBufferAsCharSequenceTest.java

```diff
@@ -61,15 +61,24 @@ private static char[] randomChars() {
         return chars;
     }
 
+    /**
+     * Randomly adjusts the position and limit such that the position will be in the first 1/4th and the limit in the last half.
+     */
     private static CharBuffer randomizeRange(CharBuffer cb) {
         int mid = cb.capacity() >>> 1;
-        int start = RAND.nextInt(mid - 3); // from 0 to mid
-        int end = RAND.nextInt(mid + 3, cb.capacity()); // from mid to capacity
+        int start = RAND.nextInt(mid >> 1); // from 0 to 1/4
+        int end = RAND.nextInt(mid + 1, cb.capacity()); // from mid to capacity
         cb.position(start);
         cb.limit(end);
         return cb;
     }
 
+    /**
+     * Generates random content to use for populating <i>cb</i> then calling through to {@code addCases(String, char[], CharBuffer, List)}
```

Comment:
Try to keep the line lengths under 100 chars (80 is better).
It makes it possible to do size-by-size reviews that some reviewers prefer.